workspace/standup/jobs.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import json
+from datetime import date, datetime, timedelta
+
+from workspace.utils import blocks
+
+
+pairs = {
+    "monday": ["Mike", "Jon"],
+    "wednesday": ["Mary", "Steve"],
+    "friday": ["Thomas", "Katie"],
+}
+
+
+def get_weekday_date(day_of_week):
+    weekday = 0  # default to Monday
+    if day_of_week == "wednesday":
+        weekday = 2
+    elif day_of_week == "friday":
+        weekday = 4
+
+    offset_days = (weekday - date.weekday(datetime.today())) % 7
```

Comment:
```suggestion
WEEKDAY_INTS = {
    "monday": 0,
    "tuesday": 1,
    "wednesday": 2,
    "thursday": 3,
    "friday": 4,
    "saturday": 5,
    "sunday": 6
    }
    
def get_next_rota_date_from_weekday(weekday_name):
    weekday_int = WEEKDAY_INTS[weekday_name.lower()]
    offset_days = (weekday_int - date.weekday(datetime.today())) % 7
```

This was the only thing I found a little less readable.

I think it's clearer if the function name refers to the future in some way. I briefly read it as "get the date of `day_of_week`", which was confusing.

I think `day_of_week` and `weekday` are very similar variables names and it's not obvious `weekday` is referring to the concept from `datetime` specifically and `day_of_week` is referring to string representations of day names.

I think just enumerating the different cases is clearer than a conditional structure and a default, and also covers future changes to the rota structure more readily.

Reply:
These are really good suggestions, thank you. I agree with you about using a dict rather than a conditional. However, there are a few places in the code where only Monday/Wednesday/Friday are supported/documented, so I only included those days for clarity (it should be obvious how to extend it for future though).